### PR TITLE
Fix enter key not submitting form

### PR DESCRIPTION
**Pull request by SonnyBA**

### CODE REVIEW

**Comment by SonnyBA** [comment]
Zie [deze commit description](https://github.com/maykinmedia/dual-listbox/pull/53/commits/6a0eef671cda4cd0aa63ce4b702ab6766e3b8c49) voor meer info

**Comment by alextreme** [comment]
Prima wat mij betreft, echter graag pas mergen na een review van Jorik

File: src/dual-listbox.js [hunk 1]
```diff
@@ -338,32 +338,34 @@ class DualListbox {
...
-        if(this.showAddButton) {
-            this.buttons.appendChild(this.add_button);
-        }
-        if(this.showRemoveButton) {
-            this.buttons.appendChild(this.remove_button);
-        }
-        if(this.showRemoveAllButton) {
-            this.buttons.appendChild(this.remove_all_button);
+
+        for (let optionName in options) {
```
Comment (JostCrow) [inline src/dual-listbox.js:359]: here is a better was of creating this loop. 

```javascript
const options = {
    this.add_all_button: this.showAddAllButton,
    ....
}

for (const [button, showButton] of Object.entries(options)) {
    button.classList.add(BUTTON_ELEMENT);
    this.buttons.appendChild(this.remove_all_button);

    if (option) {
        this.buttons.appendChild(button);
    }
}
```

Comment (SonnyBA) [reply src/dual-listbox.js:359]: We discussed this through slack earlier, resolving this comment

**Comment by SonnyBA** [comment]
Apparently Karma was causing page reloads for Safari during the tests, I could not pinpoint the exact cause but downgrading (from `6` to `5`) seems to fix it. This should be ready to be merged now.

**Review by JostCrow — APPROVED** [review]

**Pull request merged**